src/test/transaction_tests.cpp

```diff
@@ -59,10 +59,12 @@ static std::map<std::string, unsigned int> mapFlagNames = {
     {std::string("CONST_SCRIPTCODE"), (unsigned int)SCRIPT_VERIFY_CONST_SCRIPTCODE},
 };
 
-unsigned int ParseScriptFlags(std::string strFlags)
+std::vector<unsigned int> ParseScriptFlags(std::string strFlags)
```

Comment:
This is a really weird function. It'll now return a vector of:
- first item is the sum of all the flags
- then a list of the sum of all except one flag for each flag.

I think the function should be left unchanged and the client do whatever it needs with the sum of all the flags. At the very least, the function should be documented.